cmd/hyperkube/kube-proxy.go

```diff
@@ -19,24 +19,35 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
+	"os"
+
 	kubeproxy "k8s.io/kubernetes/cmd/kube-proxy/app"
 )
 
 // NewKubeProxy creates a new hyperkube Server object that includes the
 // description and flags.
 func NewKubeProxy() *Server {
-	s := kubeproxy.NewProxyServer()
+	config := kubeproxy.NewProxyConfig()
 
 	hks := Server{
 		SimpleUsage: "proxy",
 		Long: `The Kubernetes proxy server is responsible for taking traffic directed at
-		services and forwarding it to the appropriate pods.  It generally runs on
-		nodes next to the Kubelet and proxies traffic from local pods to remote pods.
-		It is also used when handling incoming external traffic.`,
-		Run: func(_ *Server, args []string) error {
-			return s.Run(args)
-		},
+	 	services and forwarding it to the appropriate pods.  It generally runs on
```

Comment:
I wasn't touching this lines, and I don't really understand what I'm supposed to do. Deleting superfluous space for now.